server/src/main/java/org/elasticsearch/index/mapper/GeoShapeFieldMapper.java

```diff
@@ -80,6 +80,8 @@ public class GeoShapeFieldMapper extends FieldMapper {
         FIELD_TYPE.setTokenized(false);
         FIELD_TYPE.setStored(false);
         FIELD_TYPE.setStoreTermVectors(false);
+        FIELD_TYPE.setDimensions(7, 4, Integer.BYTES);
+        FIELD_TYPE.setIndexOptions(IndexOptions.DOCS);
```

Comment:
Is this making the defaults explicit or changing things?

Reply:
Theses are defaults similar to `IpFieldMapper.Defaults`

Reply:
As discussed dropped the commit altogether.